subsys/partition_manager/Kconfig

```diff
@@ -74,6 +74,7 @@ endmenu # NCS subsystem configurations
 
 config PM_SINGLE_IMAGE
 	bool "Use the Partition Manager for single image builds"
+	default y if BUILD_WITH_TFM
```

Comment:
is this just a `default y` if building with TF-M, or should it actually be enforced the `PM_SINGLE_IMAGE` is selected if `BUILD_WITH_TFM=y` ?

Reply:
Enforced maybe? It doesn't work without.

Reply:
Changed it now, take a look.

Reply:
Looks good.